### ISSUE TITLE
[Issue]: Whole UI freezes when pressing prompt reuse

### ISSUE THREAD

**Issue by NilaierMusic** [vladmandic/automatic#360]
### Issue Description

When i press a prompt reuse button, no other button aside from image generation button is working until i refresh the page.

Error:
```
Traceback (most recent call last):
  File "C:\Users\nilai\Desktop\Vlad1111\automatic\venv\lib\site-packages\gradio\routes.py", line 394, in run_predict
    output = await app.get_blocks().process_api(
  File "C:\Users\nilai\Desktop\Vlad1111\automatic\venv\lib\site-packages\gradio\blocks.py", line 1078, in process_api
    data = self.postprocess_data(fn_index, result["prediction"], state)
  File "C:\Users\nilai\Desktop\Vlad1111\automatic\venv\lib\site-packages\gradio\blocks.py", line 994, in postprocess_data
    block = self.blocks[output_id]
KeyError: 331
```

### Version Platform Description

Latest commit, Windows 10, Microsoft Edge Canary

**Comment by kaelysdev** [comment]
Also got this issue multiple times when sending to txt2img or img2img from process image/image browser

**Comment by shawhu** [comment]
also, restart ui doens't fix the problem

**Comment by andyderuyter** [comment]
Same issues here. Tried from both Tab [Process image] -> Button [txt] and Tab [Image browser] -> Button [Text]

**Comment by vladmandic** [comment]
just to confirm, this button? 
![image](https://user-images.githubusercontent.com/57876960/233787032-799beb5f-4bc2-47fe-9a09-b22883388732.png)

and error happens on fresh webui start (no restart server from ui was used)?

**Comment by NilaierMusic** [comment]
> just to confirm, this button? ![image](https://user-images.githubusercontent.com/57876960/233787032-799beb5f-4bc2-47fe-9a09-b22883388732.png)
> 
> and error happens on fresh webui start (no restart server from ui was used)?

I believe that happens after the UI restarts through the button, but i've already installed a couple of extensions (like stable-diffusion-webui-state), so i don't know if that's only for me

**Comment by vladmandic** [comment]
Yes, I expect that as well. But if it happens after Restart via UI, that is a known issue and tracked separately. There is no easy way to clear Gradio in-memory states when restarting server from UI - that is going to be a long pending item. And I wrote several times about that. Any issue that happens after "Restart via UI" should be reproduced with clean restart or its not something I can look into individually.
Even issue template states that.

Closing the issue unless this can be reproduced on clean server start - if yes, I'll reopen.
